website/docs/docs/collaborate/manage-access/set-up-sso-google-gsuite.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Setting up SSO with GSuite"
-id: "setting-up-sso-with-google-gsuite"
+title: "Set up SSO with Google GSuite"
```

Comment:
If we're able to change in the URLs without too much trouble, we should change this to Google Workspace as well to reflect their branding changes.